### PR TITLE
data load for inference

### LINKED ISSUE #661: Adapt data reader to support inference (without label)

### PR DESCRIPTION
**Pull request by zhxfl**
Resolve #661

### CODE REVIEW

File: fluid/DeepASR/data_utils/async_data_reader.py [hunk 1]
```diff
@@ -31,16 +31,18 @@ class SampleInfo(object):
         label_bin_path (str): File containing the label data.
         label_size (int): Byte count of the sample's label data.
         label_frame_num (int): Label number of the sample.
+        sample_name (str): key of the sample
```
Comment (kuke) [inline fluid/DeepASR/data_utils/async_data_reader.py]: ```key``` ---> ```Key```

File: fluid/DeepASR/data_utils/async_data_reader.py [hunk 2]
```diff
@@ -102,24 +104,32 @@ def generate_sample_info_list(self):
             feature_bin_path = self._feature_bin_paths[block_idx]
             feature_desc_path = self._feature_desc_paths[block_idx]
 
-            label_desc_lines = open(label_desc_path).readlines()
             feature_desc_lines = open(feature_desc_path).readlines()
+            label_desc_lines = None
```
Comment (kuke) [inline fluid/DeepASR/data_utils/async_data_reader.py]: Seems that ```label_desc``` can be ignored?

File: fluid/DeepASR/data_utils/async_data_reader.py (hunk 1 above)
Comment (zhxfl) [reply fluid/DeepASR/data_utils/async_data_reader.py]: fix

**Review by kuke — APPROVED** [review]
LGTM

**Pull request merged**